Fixing port

furmangg opened the pull request:
Based upon the screenshot at https://docs.microsoft.com/en-us/azure/data-factory/data-movement-security-considerations I'm assuming the proper default port is 8050 not 8060. I've fixed that port in this document assuming that's correct.

azurecla commented:
Hi __@jpsnover__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@jpsnover : Thanks for your contribution to the Azure documentation! The author, @heathl17, has been notified to review your proposed change.

HeathL17 commented:
#sign-off

PRmerger commented:
Only the author of the article can use the sign off comment to add the ready-to-merge label.

Ja-Dunn commented:
@HeathL17 @jpsnover - Hi, "titlebar" should be "title bar", but this is a very minor fix that can be made at another time. Heath, if you could make a note of this to fix in a separate PR at your convenience. Otherwise, looks good to merge.

furmangg committed:
Fixing port

Based upon the screenshot at https://docs.microsoft.com/en-us/azure/data-factory/data-movement-security-considerations I'm assuming the proper default port is 8050 not 8060. I've fixed that port in this document assuming that's correct.
